docs/api-reference.md

```diff
@@ -25,20 +25,20 @@ Builds the ODBC ConnectString or JDBC Connection URL.
 **Input:** attr, a dictionary of key/value pairs
 
 ```javascript
-{"server" : "myserver.somewhere.net", "username" : "myusername"}
+{"server" : "myserver.somewhere.net"}
 ```
 
 **Return:** vector of formatted key=value pairs
 
 ```javascript
-["DRIVER={My ODBC Driver}", "UID=myusername", "Host=myserver.somewhere.net"];
+["DRIVER={My ODBC Driver}", "Host=myserver.somewhere.net"];
 ```
 
 ---
 
 ### connection-properties
 
-Similar to connection-builder but is used to build the JDBC properties file.
+Similar to connection-builder but is used to build the JDBC properties file. For JDBC Connection URL, it is suggested that connection-properties contain secure attributes such as username and password. 
```

Comment:
Maybe make it more explicit that it it is only for JDBC? Also, say that connection-properties must contain X attributes, and not suggest it. This is a requirement, not a suggestion.